src/main/java/com/sonyericsson/hudson/plugins/gerrit/trigger/config/Config.java

```diff
@@ -613,6 +615,17 @@ public void setGerritFrontEndURL(String gerritFrontEndURL) {
         this.gerritFrontEndUrl = gerritFrontEndURL;
     }
 
+    /**
+     * This method is needed for configuration as code to discover this property.
+     *
+     * It would be ignored otherwise because of the "URL" vs. "Url" naming mismatch in old getter/setter.
+     *
+     * @param gerritFrontEndURL the Url
+     */
+    public void setGerritFrontEndUrl(String gerritFrontEndURL) {
+        setGerritFrontEndURL(gerritFrontEndURL);
```

Comment:
Should the old setting be flagged as `deprecated`?

Reply:
I had no clue this option is deprecated. I am ok to leave it out from JCasC support, if that makes sense. Exposing the option through JCasC as deprecated and subject of eventual removal makes less sense to me.